lib/browser/api/dialog.js

```diff
@@ -75,6 +75,15 @@ module.exports = {
     } else if (typeof options.title !== 'string') {
       throw new TypeError('Title must be a string')
     }
+    if (options.buttonLabel == null) {
+      if (options.properties.indexOf('openDirectory') !== -1) {
+        options.buttonLabel = 'Choose'
```

Comment:
We should probably use an empty thing if it is not specified, so platform can show the default label, which is localized. Otherwise English label will always be used.